src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostUnclaimByIdControllerCFTTest.java

```diff
@@ -120,8 +126,27 @@ public void should_return_a_204_when_unclaiming_a_task_by_id() {
 
         assertions
             .taskVariableWasUpdated(taskVariables.getProcessInstanceId(), "taskState", "unassigned");
-        assertions.taskStateWasUpdatedInDatabase(taskId, "unassigned", caseworkerCredentials.getHeaders());
-        assertions.taskFieldWasUpdatedInDatabase(taskId, "assignee", null, caseworkerCredentials.getHeaders());
+
+        result = restApiActions.get(
```

Comment:
Please add this test in watasks claim function test as we are going to get rid of IAC tests